code/modules/antagonists/brother/brother.dm

```diff
@@ -99,7 +99,7 @@
 		candidates[L.mind.name] = L.mind
 
 	var/choice = input(admin,"Choose the blood brother.", "Brother") as null|anything in sort_names(candidates)
-	if(!choice)
+	if(isnull(choice))
```

Comment:
Why this change? are you trying to protect against "falsey" inputs getting ignored?

Reply:
just trying to keep with using the macros - i don't think there will be many cases where we'll get a falsey value, but it won't hurt. explosions.dm or that view range selection, maybe.

Reply:
shorter answer, i'm "being extra"

Reply:
Keep in mind, just checking for falsey rather then null is fine in most places. As an input sanity it's a good pattern (Someone somewhere will care about an empty string), but outside of that it's perfectly fine to be general